src/IECoreMaya/SceneShapeSubSceneOverride.cpp

```diff
@@ -0,0 +1,1615 @@
+{
+	IECore::V3fVectorDataPtr p( new V3fVectorData() );
+	std::vector<Imath::V3f> &pWritable = p->writable();
+	pWritable.reserve( 8 );
+
+	pWritable.emplace_back( bounds.min.x, bounds.min.y, bounds.min.z ); // Indices:
+	pWritable.emplace_back( bounds.max.x, bounds.min.y, bounds.min.z );
+	pWritable.emplace_back( bounds.max.x, bounds.max.y, bounds.min.z ); //								  7------6
+	pWritable.emplace_back( bounds.min.x, bounds.max.y, bounds.min.z ); //								 /|     /|
+	pWritable.emplace_back( bounds.min.x, bounds.min.y, bounds.max.z ); //								3------2 |
+	pWritable.emplace_back( bounds.max.x, bounds.min.y, bounds.max.z ); //								| 4----|-5
+	pWritable.emplace_back( bounds.max.x, bounds.max.y, bounds.max.z ); //								|/     |/
+	pWritable.emplace_back( bounds.min.x, bounds.max.y, bounds.max.z ); //								0------1
+
+	geometryData->positionData = p;
+
+	IECore::IntVectorDataPtr wireframeIndexData( new IECore::IntVectorData() );
+	std::vector<int> &indexWritable = wireframeIndexData->writable();
+
+	indexWritable.push_back( 0 );
+	indexWritable.push_back( 1 );
+	indexWritable.push_back( 1 );
+	indexWritable.push_back( 2 );
+	indexWritable.push_back( 2 );
+	indexWritable.push_back( 3 );
+	indexWritable.push_back( 3 );
+	indexWritable.push_back( 0 );
+	indexWritable.push_back( 4 );
+	indexWritable.push_back( 5 );
+	indexWritable.push_back( 5 );
+	indexWritable.push_back( 6 );
+	indexWritable.push_back( 6 );
+	indexWritable.push_back( 7 );
+	indexWritable.push_back( 7 );
+	indexWritable.push_back( 4 );
+	indexWritable.push_back( 7 );
+	indexWritable.push_back( 3 );
+	indexWritable.push_back( 6 );
+	indexWritable.push_back( 2 );
+	indexWritable.push_back( 4 );
+	indexWritable.push_back( 0 );
+	indexWritable.push_back( 5 );
+	indexWritable.push_back( 1 );
+
+	geometryData->wireframeIndexData = wireframeIndexData;
+
+	// \todo: For completeness we should probably create data for rendering a shaded box at some point
+	geometryData->normalData.reset( new IECore::V3fVectorData() );
+	geometryData->uvData.reset( new IECore::V2fVectorData() );
+	geometryData->indexData.reset( new IECore::IntVectorData() );
+}
+
+void fillPointsData( ConstObjectPtr object, GeometryDataPtr geometryData )
+{
+	IECoreScene::ConstPointsPrimitivePtr pointsPrimitive = IECore::runTimeCast<const IECoreScene::PointsPrimitive>( object );
+	assert( pointsPrimitive );
+
+	IECore::ConstV3fVectorDataPtr p = pointsPrimitive->variableData<IECore::V3fVectorData>( "P" );
+	geometryData->positionData = p;
+
+	geometryData->normalData.reset( new IECore::V3fVectorData() );
+	geometryData->uvData.reset( new IECore::V2fVectorData() );
+	geometryData->indexData.reset( new IECore::IntVectorData() );
+	geometryData->wireframeIndexData.reset( new IECore::IntVectorData() );
+}
+
+void fillCurvesData( ConstObjectPtr object, GeometryDataPtr geometryData )
+{
+	// \todo: currently a curve is built from linear segments. Needs proper interpolation at some point.
```

Comment:
That's correct. I've tested with a few files and found the visual difference to be fairly small, but yeah, it's definitely ignoring the basis at the moment.

Reply:
I'm happy to leave it until we get user feedback, but here are some example curves that I expect will look quite odd if we just draw linear:

```
/data/share/andrewk/curve.cob
/data/share/andrewk/curves.cob
```

Reply:
Oh yes, quite odd indeed.

Reply:
@murrays-ie and company didn't seem concerned about this for a first pass, since our grooms usually have enough cvs for linear not to be a distraction.

Reply:
Great, thanks for checking with them!